ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -31,20 +38,20 @@ No prerequisites to fulfill.
 
 3. Enter the desired number of workers for the queue type in the *Number of workers* field.
 
-    > [Info] A worker is a job executing the actions within a process. One worker can only execute one action at the same time. By assigning multiple workers to a queue type, this queue type may execute several actions at the same time. Consequently, actions with a queue type with more workers are executed faster that actions with queue types with fewer workers. The total number of workers of all queue types depends on the number of booked vCores.   
-    For one worker, 4 vCores are needed. Additionally, 10 % of the booked vCores, but at least 4 vCores are blocked for the daily business and cannot be used for workers. The number of vCores needed always refers to the total number of workers in all queue types. For example, for a queue type with 2 workers and a queue type of 4 workers, 28 vCores are needed:   
-    *4 vCores per worker (4 \* 6 = 24) plus 4 vCores blocked for the daily business (24 + 4 = 28)*
-
 4. Click the *Execution priority* drop-down list and select the appropriate execution priority. The following options are available:
     - **Oldest executable action**   
         The oldest executable actions within all existing processes is executed first by the workers.
     - **Executable actions of oldest process**   
         The executable actions within the oldest process are executed first by the workers.
 
-5. Click the [SAVE] button in the upper right corner.   
-    The *Create queue type* view is closed. The new queue type has been created and is displayed in the list of queue types. The *Queue type has been created* pop-up window is displayed.
+5. If you want to create a queue type to throttle a process, enable the ![Throttling](../../Assets/Icons/Toggle.png "[Throttling]") (Throttling) toggle.   
```

Comment:
In Integration/Operation schreiben wir toggle anders. Toggle-Symbol verwenden wir nur in User interface Kapitel. S. unten (Beispiele aus WFII): 
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/fc50d919-7e46-47a1-8509-f003cccc4128)
vs. 
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/bff3aef4-071e-42f8-a405-71f2555dc5a1)